rfcs/20250618-pr-rules/README.md

```diff
@@ -0,0 +1,207 @@
+aligning the correct reviewers for a PR (setting up meetings, collecting
+information, etc.).
+16. The person who merges a PR is expected to understand the consequences,
+especially with respect to approvals and review (e.g. a single approval may
+not be sufficient for merge).
+17. The reviewer should mark comments as resolved unless PR author directly
+addresses their comment with code change
+19. To avoid overuse of "will save for follow-up PR"/"not in the scope of PR"
+the author is responsible resolving with reviewer the follow up steps in
+detail.
+20. Changes to these rules must be proposed following the RFC process.
+21. Additional private CI runs can be requested.
+22. While oneDAL is open source, hardware IP should be protected especially in
+public forums like Slack and GitHub.
+23. An admin can wipe a review or otherwise change a PR as necessary. (Reviewer
+unavailable to re-review, etc.)
+24. Description must describe the change and reasoning behind it.
+25. These are not hard and fast and can be changed in cases that warrant it.
+26. When merging, use the title of the PR and not of the commit(s).
+
+## Section 3: Bug Fix
+
+1. PR Title must include the associated algorithm or oneDAL code feature and
+the name of the bug.
+2. Description must describe the issue observed and how the fix solves the
+bug/issue.
+3. If the bug fix is associated with an issue raised on Github, it must be
+linked.
+4. If the bug fix solves the Github issue and is merged, the Github issue
+should be closed.
+5. Fixes which solves a CI failure must be listed for verification by the
+reviewer.
+
+## Section 4: Dependencies
+
+1. Dependency PRs not handled by automation should be handled per the
+guidance of reviewer with knowledge of the change (e.g. code owner).
+2. Questions about dependency changes can/should be addressed to the team at
+large.
+
+## Section 5: Documentation
+
+1. The PR title must include the related feature and/or DAAL/oneDAL component.
+2. The documentation must follow DAAL or oneDAL documentation guidelines.
+
+## Section 6: Feature
+
+1. Features which change (not just add to) the user-facing API or ABI or change
+dependencies in a meaningful way should have an RFC (see section 10)
+2. Features in oneDAL must follow oneDAL conventions (especially for API and
+structure).
+3. Features in DAAL must follow DAAL conventions.
+4. New features must have associated new tests.
+
+## Section 7: Infrastructure
+
+1. As infra has wide-ranging impact, code review should be stricter and closely
+reviewed by those knowledgeable.  
+2. Those which impact a specific ISA should have a representative with domain
+knowledge review those changes.
+
+## Section 8: Performance
+
+1. Performance benchmarks must include scikit-learn_bench\*\*\*\*\* if
+the algorithm is included or to be included in scikit-learn-intelex.
+2. Performance benchmarks must be provided to reviewers via email.
```

Comment:
Is this unnecessary ? i think any channel would work, even if results would be shared within PR itself